themes/app/src/scss/_footer.scss

```diff
@@ -0,0 +1,12 @@
+img {
+    width: 100px;
+    height: auto;
+}
+
+.footer-partners {
```

Comment:
I'm not sure if this is being used as your template is using `footer__partners`, might need to be updated :)